plonetheme/barceloneta/theme/less/header.plone.less

```diff
@@ -59,19 +59,20 @@
         max-width: 50%;
         padding-top: 0;
         position: relative;
-        width: 250px;
+        width: 260px;
```

Comment:
Without looking into details, why does the searchbox width need to specified here? Leaving it out we have better room for automatically growing the submit button.